### DISCUSSION TITLE
Set ImageBrush much more simple?

### DISCUSSION THREAD

**Discussion by Flithor** [AvaloniaUI/Avalonia#11895] [Ideas]
Current
```
<UserControl>
  <UserControl.Background>
    <ImageBrush Source="/Assets/Bg.png"/>
  </UserControl.Background>  
</UserControl>
```
What I want:
```
<UserControl Backgound="image(/Assets/Bg.png)"/>
```
like set transform: `RenderTransform="matrix(-1,0,0,-1,0,0)"`

**Comment by timunie** [comment]
I think this is an interesting idea, however I see some issues here:
- No binding possible which is quite common
- No async image loading possible which can result in a blocking of the UI if the image is huge
- I see no way to control other parameters easily like `Stretch`, `StretchDirection` etc which are also commonly used.

**Reply by cesarchefinho** [reply]
Write an MarkupExtension and use

< UserControl Backgound="{MyMarkupImgBind /Assets/Bg.png}" / >

Also MyMarkupImgBind can have Async and FalbackImage,  Stretch, StretchDirection paramryrts etc
